### PR TITLE
Qos manager framework

### PR DESCRIPTION
**Pull request by stormgbs**
### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/common/utils/containers.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+
+	"github.com/koordinator-sh/koordinator/pkg/util"
+	"github.com/koordinator-sh/koordinator/pkg/util/runtime"
+)
+
+// KillContainers kills containers inside the pod
+func KillContainers(pod *corev1.Pod, message string) {
```
Comment (jasonliu747) [inline pkg/koordlet/common/utils/containers.go:28]: should we move `KillContainers` under `pkg/utils/container`?

File: pkg/koordlet/common/utils/pod.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+package utils
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/koordinator-sh/koordinator/pkg/koordlet/statesinformer"
+	"github.com/koordinator-sh/koordinator/pkg/util"
+)
+
+func GetPodMetas(pods []*corev1.Pod) []*statesinformer.PodMeta {
```
Comment (jasonliu747) [inline pkg/koordlet/common/utils/pod.go:26]: Should we move `GetPodMetas` under `pkg/utils/pod`?

File: pkg/koordlet/common/utils/containers.go (hunk 1 above)
Comment (hormes) [reply pkg/koordlet/common/utils/containers.go:28]: It's better to move when needed

**Pull request merged**